src/shopilingus/Product.java

```diff
@@ -5,37 +5,82 @@
     private static int nextId = 0;
 
     public int id;
-    private double price;
+    public double price;
```

Comment:
when you have getters and setters then all your class fields should be private and in other classes you should use public getters and setters instead of accessing fields